src/main/docker/Dockerfile.jdk

```diff
@@ -16,7 +16,48 @@
 
 # Docker registry
 ARG DOCKER_HUB_PUBLIC=docker.io
-FROM ${DOCKER_HUB_PUBLIC}/cafapi/opensuse-base:2
+
```

Comment:
I'm still a bit unclear here on how this works.

**Q1)**

I should not update the release notes or make this a breaking change because this is a temporary change, is that correct? I noticed we did not update release notes when we did this [previously](https://github.com/CAFapi/opensuse-java17-images/commit/76b7a367f304c3bd7e96a2a65420673ab599e80c ).

**Q2)**

Since this repo has a lot of downstream dependencies:

http://16.103.38.246:9090/search?full=%22opensuse-jre11%22&defs=&refs=&path=&hist=&type=&xrd=&nn=11&si=full&searchall=true&si=full

![image](https://user-images.githubusercontent.com/55391479/196470341-ff6dcf96-8565-4f06-a733-6e15827179b7.png)

I'm worried that these changes may break a lot of consumers unrelated to this ticket?

**Q3)**

I'm not sure how long it takes for LEAP to get these changes in Tumbleweed? If we do a FAS release before that happens, do we just release the way it is here, using Tumbleweed?

Reply:
As Michael says, these changes are fine for testing but do not merge them.